recipes/hmftools-mark-dups/1.1.5/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+[ -d "$TGT" ] || mkdir -p "$TGT"
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+
+cd "${SRC_DIR}"
```

Comment:
I think this is no-op. You are already in this folder.

Reply:
The line could be removed but it does not do any harm either!

Reply:
Got it, I'll leave it untouched for fear of breaking something! And thanks for the review